scaladoc/src/dotty/tools/scaladoc/snippets/WrappedSnippet.scala

```diff
@@ -4,11 +4,11 @@ package snippets
 import java.io.ByteArrayOutputStream
 import java.io.PrintStream
 
-case class WrappedSnippet(snippet: String, lineOffset: Int, columnOffset: Int)
+case class WrappedSnippet(snippet: String, lineOffset: Int, columnOffset: Int, lineBoilerplate: Int, columnBoilerplate: Int)
 
 object WrappedSnippet:
-  private val lineOffset = 2
-  private val columnOffset = 2
+  private val lineBoilerplate = 2
```

Comment:
:laughing: I liked the offset names better.

Reply:
Yes, me too, but there's already lineOffset and columnOffset param in method signature and I didn't want problems with obscured variables.

Reply:
fair enough

Reply:
Boilerplate does not make any sense. Maybe we should extend the name to get `fileLineOffset` `docstringLineOffset` or something like that. It would be self explanatory then.

Reply:
I like sound of `lineBoilerplate` or better `The line of boilerplate` (would make nice blogpost or novel title) but I agree with @KacperFKorban and @BarkingBad that is not good name for value

Reply:
I've changed that to outerOffset which is offset from docstring to snippet and innerOffset which is offset from start of the snippet to body